app/site/questions.js

```diff
@@ -12,6 +12,7 @@ import TaxiMostCommonDropoff from '../components/questions/TaxiMostCommonDropoff
 import TaxiPaymentMethod from '../components/questions/TaxiPaymentMethod'
 import EscooterCity from '../components/questions/EscooterCity'
 import CovidCasesPerRideshare from '../components/questions/CovidCasesPerRideshare'
+import RideTrips from '../components/questions/RideTripsOhare'
```

Comment:
Can we rename this to something like `OHareRideshares`? Both the component you export and the file should have the same name.